doc/mainpage.dox

```diff
@@ -84,6 +84,11 @@ in a property tag (or any tag for a special property, see Secion \ref special_pr
                   writeable.
 - `macro_pulse_number_source`: Name of process variable which contains the macro pusle number which should be attached
                                to the properties.
+- `persist`: Controls behaviour of DOOCS persistency files. This is only important for writable arrays with more than MAX_CONF_LENGTH entries 
+             (usually 20). Possible values are `true` (the default) - always save the array,
+             `false` -  do not save the array,
+             `auto` - default DOOCS behaviour, only arrays with up to 20 entries are saved, in server config file.
+             Long arrays are saved in separate files below `hist/` instead of the server config file.
```

Comment:
The context of the last sentence seems to be ambiguous to me. It does not refer to the `auto` option, right (with `auto` arrays are never stored in a separate file, as only short arrays which fit into the config file are stored)? Maybe moving it up before the enumeration of possible options.

Reply:
fixed.